ToolsTest/VasylTests/xUnit.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Tools.Models;
+using Xunit;
+
+namespace ToolsTest.VasylTests
+{
+    public class xUnit
+    {
+        [Fact]
+        public void EmailProperty_CanBeSetAndRetrieved()
+        {
+            // Arrange
+            string expectedEmail = "test@example.com";
+            User user = new User("John");
+
+            // Act
+            user.Email = expectedEmail;
+            string actualEmail = user.Email;
+
+            // Assert
+            Assert.Equal(expectedEmail, actualEmail);
```

Comment:
You do not need to test the auto-property (like here - `Email`) as factually you just test the Microsoft .Net Framework and CLR instead of the program code. You can test it ONLY if the property should have the custom value but not default.
So, this test will always return true and this fact makes it a non-test.

Better for testing here is to test that property is set when you provide an email instancy by constructor.

```suggestion
            // Arrange
            string expectedEmail = "test@example.com";

            // Act
            User user = new User("John", expectedEmail);

            // Assert
            Assert.Equal(expectedEmail, user.Email);
```